back/spec/acceptance/ideas_spec.rb

```diff
@@ -1189,6 +1198,29 @@
               end
             end
 
+            context 'Moving the idea from a voting phase' do
+              let(:project) { create(:project_with_past_ideation_and_active_budgeting_phase) }
+              let(:idea) { create(:idea, project: project, phases: [project.phases.last]) }
+              let(:id) { idea.id }
+
+              context 'Moving between phases' do
+                let(:phase_ids) { [project.phases.first.id] }
+
+                example_request 'Move the idea from a voting phase' do
```

Comment:
Do we want to include this example and the next in the docs?

Reply:
No probably not needed, will add document: false